scripts/postCss.js

```diff
@@ -14,44 +11,11 @@ cssFiles.map(name => fs.readFile(`src/${name}.css`, (err, css) => {
     .use(comments({
       remove: function(comment) { return comment[0] == "@"; }
     }))
-    .use(removeRules({
-      rulesToRemove: {
-        ':root': '*'
-      }
-    }))
     .process(css, { from: `src/${name}.css`, to: `src/${name}.css` })
     .then(result => {
       fs.writeFile(`src/${name}.css`, result.css, () => true)
       if ( result.map ) {
-        fs.writeFile(`src/${name}.map`, String(result.map), () => true)
+        fs.writeFile(`src/${name}.map`, result.map.toString(), () => true)
```

Comment:
Why is this changed from the standard in the Generator?

Reply:
this one also not sure, I assume my IDE on save linters did this, reverting back.